api/app/tests/test_health.py

```diff
@@ -0,0 +1,37 @@
+""" Test the health endpoint
+"""
+import os
+import json
+import requests
+from starlette.testclient import TestClient
+import app.main
+from app.tests.common import MockRequestResponse
+
+
+def test_health_ok():
+    """ Test health endpoint, given that everything is fine """
+    client = TestClient(app.main.app)
+    response = client.get('/health/')
+    assert response.status_code == 200
```

Comment:
The 200 check is redundant - you can remove it (I prefer also only having one assertion per test)